commands/deploy.go

```diff
@@ -326,3 +324,14 @@ func deriveFprocess(function stack.Function) (string, error) {
 func languageExistsNotDockerfile(language string) bool {
 	return len(language) > 0 && strings.ToLower(language) != "dockerfile"
 }
+
+func compileSecrets(cmdLineSecrets []string, yamlSecrets []string) []string {
+	var merged []string
```

Comment:
We should make sure we first take all the YAML Then overlay (replace) any keys found in the CLI / cmd-line args. We don't want to have duplicates/conflicts.